connector/connect/common/src/main/protobuf/spark/connect/commands.proto

```diff
@@ -65,6 +65,12 @@ message SqlCommand {
 
   // (Optional) A sequence of literal expressions for positional parameters in the SQL query text.
   repeated Expression.Literal pos_args = 3;
+
+  // (Optional) A map of parameter names to expressions.
+  map<string, Expression> arguments = 4;
+
+  // (Optional) A sequence of expressions for positional parameters in the SQL query text.
```

Comment:
shall we doc that this can't coexist with `named_arguments`?

Reply:
shall we also mention that the `args` and `pos_args` are deprecated?

Reply:
@grundprinzip @HyukjinKwon Is there any way to deprecate some fields?

Reply:
Would this be enough `[deprecated=true]`?